example02/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist.js
```

Comment:
In ac3cbf2 I made it such that examples don't need their own .gitignore files, so there's no need to add one here.

Reply:
Alright. I'll remove the gitignore file.